s3response/s3response.go

```diff
@@ -283,30 +283,30 @@ type Bucket struct {
 
 type ListBucketsInput struct {
 	Owner             string
-	IsAdmin           bool
 	ContinuationToken string
 	Prefix            string
 	MaxBuckets        int32
+	IsAdmin           bool
 }
 
 type ListAllMyBucketsResult struct {
 	XMLName           xml.Name `xml:"http://s3.amazonaws.com/doc/2006-03-01/ ListAllMyBucketsResult" json:"-"`
 	Owner             CanonicalUser
-	Buckets           ListAllMyBucketsList
 	ContinuationToken string `xml:"ContinuationToken,omitempty"`
 	Prefix            string `xml:"Prefix,omitempty"`
+	Buckets           ListAllMyBucketsList
```

Comment:
order probably doesnt matter here, but do we want to try to keep with similar filed order to aws?